### ISSUE TITLE
Use enzyme instead of react-test-renderer

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#671]
**Is your feature request related to a problem? Please describe.**
Currently we use the library `react-test-renderer` for our unit tests. There have been various times where using `enyzme` would have given more flexibility in testing with the most recent case being that `react-test-render` and `react-lazyload` do not seem to work together ([details here](https://github.com/bbc/simorgh/pull/581#issuecomment-422412788)). 

**Describe the solution you'd like**
We should remove `react-test-renderer` and replace it with a combination on `enzyme` and `enzyme-to-json`. This should require the initial setup is to add this to the jest config in the package.json

```
"jest": {
    "snapshotSerializers": [
      "enzyme-to-json/serializer"
    ]
}
```

Then an update the Test utility functions `/src/app/helpers/tests/testHelpers.js` to use `enzyme-to-json`

**Describe alternatives you've considered**
N/A

**Comment by pjlee11** [comment]
This is blocked by https://github.com/bbc/simorgh/issues/671

**Comment by ChrisBAshton** [comment]
Superseded by https://github.com/BBC-News/psammead/issues/219
